### PR TITLE
Implement Character::Skills endpoint.

### PR DESCRIPTION
**Pull request by amclain**
For #69

### CODE REVIEW

**Comment by amclain** [comment]
@tslogan ready for review.

File: lib/greeve/character/skills.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      endpoint "char/Skills"
+
+      attribute :free_skill_points, xpath: "eveapi/result/freeSkillPoints/?[0]", type: :integer
+
+      rowset :skills, xpath: "eveapi/result/rowset[@name='skills']" do
+        attribute :type_id,     xpath: "@typeID",      type: :integer
+        attribute :type_name,   xpath: "@typeName",    type: :string
+        attribute :skillpoints, xpath: "@skillpoints", type: :integer
+        attribute :level,       xpath: "@level",       type: :integer
+        attribute :published,   xpath: "@published",   type: :integer
```
Comment (tslogan) [inline lib/greeve/character/skills.rb]: Should the `published` attribute type be `boolean` instead? It looks like the documentation might be wrong:
https://eveonline-third-party-documentation.readthedocs.io/en/latest/xmlapi/character/char_skills.html

Comment (amclain) [reply lib/greeve/character/skills.rb]: Good point. I've changed the type to `boolean`.

**Comment by tslogan** [comment]
:shipit:

**Pull request merged**